src/pyramid_googleauth/services/signature.py

```diff
@@ -0,0 +1,47 @@
+import random
```

Comment:
This is copy pasted from checkmate

I don't think it belongs in this package and it probably too small for it's own extension